app/views/promotion/landingPage.scala.html

```diff
@@ -1,12 +1,13 @@
 @import com.gu.memsub.promo.PromoCode
 @import com.gu.memsub.promo.Promotion.asAnyPromotion
-@import com.gu.memsub.promo.Promotion.AnyPromotionWithLandingPage
+@import com.gu.memsub.promo._
 @import com.gu.subscriptions.DigipackCatalog
 @import model.DigitalEdition
 @import views.support.Dates._
 @import views.support.LandingPageOps._
+@import scalaz.Id._
 @import org.joda.time.Days
-@(edition: DigitalEdition, catalog: DigipackCatalog, promoCode: PromoCode, promotion: AnyPromotionWithLandingPage, defaultTrial: Days)
+@(edition: DigitalEdition, catalog: DigipackCatalog, promoCode: PromoCode, promotion: Promotion[PromotionType, Id, SubscriptionsLandingPage], defaultTrial: Days)
```

Comment:
Does this need to have both Id and SubscriptionsLandingPage in the type, doesn't the latter type imply the landing page's definite presence?